app/services/editions/publishing/Publishing.scala

```diff
@@ -0,0 +1,100 @@
+package services.editions.publishing
+
+import java.time.OffsetDateTime
+import com.gu.pandomainauth.model.User
+import logging.Logging
+import model.editions.Edition.{FeastNorthernHemisphere, FeastSouthernHemisphere}
+import model.editions.{EditionsIssue, PublishAction}
+import net.logstash.logback.marker.Markers
+import services.editions.db.EditionsDB
+import play.api.libs.json.Writes
+
+import scala.jdk.CollectionConverters._
+
+class Publishing[+A:Writes, +B:Writes](editionsAppPublicationBucket: PublicationTargetBase[A],
+                 editionsAppPreviewBucket: PublicationTargetBase[A],
```

Comment:
Could these be, concretely, `EditionsBucket`, `EditionsBucket`, and `FeastPublicationTarget`? Struggling to see how these type params will be exercised in a useful way, but may be missing something!

Reply:
Yes, they could TBH